examples/prometheus/rules/kube.rules

```diff
@@ -0,0 +1,68 @@
+groups:
+- name: kubernetes-rules
+  rules:
+
+    - alert: DockerLatencyHigh
+      expr: max(kubelet_docker_operations_latency_microseconds{quantile="0.9"}) / 1e+06 > 1
+      for: 5m
+      labels:
+        severity: warning
```

Comment:
Is there a reason for the severity being a label and not an annotation?
It's part of the alert definition metadata and not the metric that generated it.

Reply:
Having it as a label allows the alert to be routed to different receivers in AlertManager.

Reply:
> Having it as a label allows the alert to be routed to different receivers in AlertManager.

That's my understanding ([upstream docs](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/)). We could also add as an annotation if we determine it's useful for in some way.

Reply:
Hmm make sense.
It also means that if you did not define a severity on your alert and a generating expression has that label it will be set from that (I'm not sure what the precedence is if both exist).